src/host_bsp.c

```diff
@@ -458,6 +467,16 @@ void _host_sync() {
     }
 }
 
+void _microsleep(int microseconds)
+{
+    // This function itself is about 300 microseconds overhead
```

Comment:
Seems like an excessive amount, does this mean if you call _microsleep(1) it actually sleeps for 301 microseconds?